docs/index.yaml

```diff
@@ -3,7 +3,26 @@ entries:
   promitor-agent-resource-discovery:
```

Comment:
Can you revert this please? I'll do a release myself later on.

Reply:
Done. I've did that according to CONTRIBUTING.md doc :)

Reply:
My bad, I should remove that